chwifi

```diff
@@ -161,22 +156,19 @@ connect_dispatch() {
         # Change work profile password and do connection routine
         change_profile_password "$network_manager_work_profile" "$daily_password"
         connection_routine "$network_manager_work_profile"
-    elif check_profile "$1" ; then
+    else
+        # If not work, simply connect to given profile
         printf '%s\n' "Connecting to $1"
         connection_routine "$1"
-    else
-        printf "An error occured while connecting to: '%s', please check" "$1"
-        exit 1
     fi 
 }
 
 display_help() {
     printf "Usage: chwifi [OPTION]... <PROFILE>\n"
-    printf "Connect to home or work wireless networks, caching rolling passwords at work\n\n"
+    printf "Connect any network-manager wireless networks, caching rolling passwords at work\n\n"
```

Comment:
```suggestion
    printf "Connect to any netctl wireless networks, caching rolling passwords at work\n\n"
```

You forgot 'to' and let's keep the help-message specific until broader support is implemented.